bees/emailbee/emailbeefactory.go

```diff
@@ -116,9 +116,13 @@ func (factory *EmailBeeFactory) Actions() []bees.ActionDescriptor {
 				},
 				{
 					Name:        "text",
-					Description: "Content of the email",
+					Description: "Content of the email using plain text",
+					Type:        "string",
+				},
+				{
+					Name:        "html",
+					Description: "Content of the email using HTML",
 					Type:        "string",
-					Mandatory:   true,
```

Comment:
Actually, we could even make neither Mandatory and allow empty emails with just a subject. I've seen people do that (for better or worse).